src/templates/postTemplate.scss

```diff
@@ -63,7 +63,7 @@ div.post-page {
         margin-top: 0;
         padding: 20px;
         border-radius: 10px;
-        background-color: #fff7ea;
+        background-color: #e9e7f1;
```

Comment:
why're we changing stylesheets in a blog post?

Reply:
This is NOT really the right place for stylesheet changes, but because the website is not super mature, a few things looked odd:
- lack of margins between list items,
- images placed too close to text inside list items,
- yellowish blockquote color not matching this year's navy/purplish theme.
For the last one:

| | Before | After |
| --- | --- | --- |
| Blog | <img width="590" alt="Blog before" src="https://user-images.githubusercontent.com/4550621/128710386-23649f06-e0a4-4731-99be-842794efcf36.png"> | <img width="587" alt="Blog after" src="https://user-images.githubusercontent.com/4550621/128710392-57103fd1-b1c1-43da-b384-bbb37dfcb864.png"> |
| Docs | <img width="601" alt="Docs before" src="https://user-images.githubusercontent.com/4550621/128710399-eba5d907-7e6e-48e5-a6bc-dc56c2838720.png"> | <img width="602" alt="Docs after" src="https://user-images.githubusercontent.com/4550621/128710396-f67bdfa6-2a89-4e80-a042-0d0442c4f700.png"> |